pages/profiles/navneetkarnam.md

```diff
@@ -0,0 +1,36 @@
+# NAVNEET KARNAM
+- Location: NEW PALTZ, NY
+- TimeZone: Eastern Standard Time
+- Operating System: Mac OS Sierra
+* * *
+## Summary: 
+> Strong information technology professional with a Master of Science (M.S.) focused in Computer Science from State University of New York at New Paltz. Experienced Information Technology Technician with a demonstrated history of working at the IT Help desk State University of New York at New Paltz. Skilled in Debugging Code, Java,JavaFX, HTML, CSS, AngularJs, MySQL and Arduino
+* * *
+
```

Comment:
Why do you creating many blank lines? I think the markdown will not be rendered?